app/assets/stylesheets/common/base/user.scss

```diff
@@ -557,16 +557,6 @@
   }
 
   &.second-factor-backup-preferences {
-    padding-left: 0;
```

Comment:
I'm unclear if it's part of this PR or something you fixed while doing it? if it's not exacty part of the PR, you can leave it or make another commit if you have time, it would be a little bit more cleaner.